models/benchmarks/lulesh_model.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+from models.benchmarks.benchmark_model import BenchmarkModel
+import os
+
+class BenchmarkModelImplementation(BenchmarkModel):
+    """This class is an implementation of the BenchmarkModel for LULESH"""
+
+    def __init__(self):
+        super().__init__()
+        self.name = 'lulesh'
+        self.base_runflags = ''
+        self.base_compileflags = '-DUSE_MPI=0 -fopenmp'
+        self.base_linkflags = '-O3 -fopenmp'
+        self.base_build_deps = ''
+        self.base_run_deps = ''
+        self.benchmark_url = 'https://github.com/BaptisteGerondeau/LULESH.git'
+
+    def prepare_build_benchmark(self, extra_deps):
+        """Prepares Environment for building and running the benchmark
+        This entitles : installing dependencies, fetching benchmark code
+        Can use Ansible to do this platform independantly and idempotently"""
+        prepare_cmds = []
```

Comment:
you can initialise this directly as a list of lists

Reply:
We are talking about prepare_cmds = [[]] right ?